src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+    private static final int lotConst = 100;

```

Comment:
Uppercase for constant names
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
![image](https://user-images.githubusercontent.com/20326858/224543238-6521042b-bd0e-4fb2-8b31-8a2796555f61.png)